jdaviz/configs/specviz/plugins/line_analysis/tests/test_lineflux.py

```diff
@@ -12,9 +12,9 @@
 
 # Maps input spectrum flux unit to expected line analysis flux unit
 expected_lineflux_results = {
-    u.Jy/u.sr: u.Unit('W/(m2*sr)'),
+    u.Jy/u.sr: u.Unit('W/(m2)'),
     u.Jy: u.Unit('W/m2'),
-    u.Unit('W/m2/m')/u.sr: u.Unit('W/(m2*sr)'),
+    u.Unit('W/m2/m')/u.sr: u.Unit('W/(m2)'),
     u.Unit('W/m2/m'): u.Unit('W/m2')
```

Comment:
Hopefully these changes make sense in the context of the sum spectrum using units without per steradian.